deepchem/models/torch_models/infograph.py

```diff
@@ -303,10 +303,10 @@ class InfoGraphModel(ModularTorchModel):
 
     Parameters
     ----------
-    num_features: int
-        Number of node features for each input
-    edge_features: int
-        Number of edge features for each input
+    node_fdim: int
```

Comment:
I prefer num_features over node_fdim I think. I'm not sure about this renaming. Let's discuss offline